test/integration/users_login_test.rb

```diff
@@ -0,0 +1,78 @@
+require 'test_helper'
+
+class UsersLoginTest < ActionDispatch::IntegrationTest
```

Comment:
If this is written by you and not generated, you could try to rewrite it using RSpec